test/components/collapse.test.ts

```diff
@@ -40,6 +40,7 @@ describe('o-collapse', () => {
     expect(collapseItemWrappers[0].vm.title).toBe('title1')
     expect(collapseItemWrappers[1].vm.disabled).toBe(true)
 
+    // there cant get `isActive`
     expect(collapseItemWrappers[0].vm.isActive).toBe(true)
```

Comment:
In here, I cant get the `isActive` 🤔

Reply:
Yes, some components are having problems with their test files, but that's all going to be rewritten.